### ISSUE TITLE
`conda-env-deactivate` is broken by recent changes to `conda-env-stripped-path`

### ISSUE THREAD

**Issue by WJCFerguson** [necaris/conda.el#32]
I just updated (;; Package-Version: 20190528.421) and can't activate an environment now.

`conda-env-stripped-path` now tries to split a PATH string into a list of strings, but it is being fed `exec-path` in `conda-env-deactivate`, which is already a list of strings.

```
Debugger entered--Lisp error: (wrong-type-argument stringp ("/home/james/bin" "/usr/local/sbin" "/usr/local/bin" "/usr/sbin" "/usr/bin" "/sbin" "/bin" "/usr/games" "/usr/local/games" "/snap/bin" "/usr/local/libexec/emacs/26.2/x86_64-pc-linux-gnu"))
  string-match(":" ("/home/james/bin" "/usr/local/sbin" "/usr/local/bin" "/usr/sbin" "/usr/bin" "/sbin" "/bin" "/usr/games" "/usr/local/games" "/snap/bin" "/usr/local/libexec/emacs/26.2/x86_64-pc-linux-gnu") 0)
  split-string(("/home/james/bin" "/usr/local/sbin" "/usr/local/bin" "/usr/sbin" "/usr/bin" "/sbin" "/bin" "/usr/games" "/usr/local/games" "/snap/bin" "/usr/local/libexec/emacs/26.2/x86_64-pc-linux-gnu") ":" nil)
  (progn (split-string s separator omit-nulls))
  (unwind-protect (progn (split-string s separator omit-nulls)) (set-match-data save-match-data-internal (quote evaporate)))
  (let ((save-match-data-internal (match-data))) (unwind-protect (progn (split-string s separator omit-nulls)) (set-match-data save-match-data-internal (quote evaporate))))
  s-split(":" ("/home/james/bin" "/usr/local/sbin" "/usr/local/bin" "/usr/sbin" "/usr/bin" "/sbin" "/bin" "/usr/games" "/usr/local/games" "/snap/bin" "/usr/local/libexec/emacs/26.2/x86_64-pc-linux-gnu"))
  conda-env-stripped-path(("/home/james/bin" "/usr/local/sbin" "/usr/local/bin" "/usr/sbin" "/usr/bin" "/sbin" "/bin" "/usr/games" "/usr/local/games" "/snap/bin" "/usr/local/libexec/emacs/26.2/x86_64-pc-linux-gnu"))
  conda-env-deactivate()
  conda-env-activate()
  funcall-interactively(conda-env-activate)
  call-interactively(conda-env-activate record nil)
  command-execute(conda-env-activate record)
  #f(compiled-function (cmd) #<bytecode 0x1b4de25>)("conda-env-activate")
```

**Comment by necaris** [comment]
@WJCFerguson great catch, I'll try and get a fix out today, thank you for the report!

**Comment by necaris** [comment]
@WJCFerguson this should be closed by #33 and the updated build should be on MELPA in a few hours, please let me know if things didn't work as expected!

**Comment by amontz** [comment]
@necaris Thanks for the quick fix, but unfortunately with the 20190531.1807 release I am still not able to activate an environment. `conda-env-activate` now gives a `Wrong type argument: listp` error.

**Comment by fleimgruber** [comment]
I hit that one too. `conda-env-stripped-path` now expects a list of strings, not `exec-path` and fails in https://github.com/necaris/conda.el/blob/master/conda.el#L265
